docs/src/library-csharp.md

```diff
@@ -56,6 +56,21 @@ await using var browser = await playwright.Firefox.LaunchAsync(new()
 });
 ```
 
+## Using Assertions
+
+You can do the following to leverage Playwright's assertions when you are using your own test framework:
```

Comment:
It needs a bit more context describing why the assertions are useful.

Reply:
done